pkg/scripts/csv-to-couchdb.ts

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env -S deno run --allow-read --allow-net
+
+import Denomander from "https://deno.land/x/denomander@0.9.1/mod.ts";
+import { readCSV } from "https://deno.land/x/csv/mod.ts";
+import ProgressBar from "https://deno.land/x/progress@v1.2.4/mod.ts";
+import { BufReader } from "https://deno.land/std/io/buffer.ts";
+import { bulkDocs, createDatabase } from "../couchdb/couchdb.ts"
+
+/**
+  * Counts lines in a csv file for progress bar calculations
+  * @param {string} filePath - csv file path 
+  * @return {Number} - Number of lines in file
+  */
+
+const howManyLines = async function (filePath: string) {
+    let lineCount: number = 0;
+    var file;
+    console.log({ filePath })
```

Comment:
This can be removed. Also, why print
```
{
    "filepath": "path/to/the/csv_file.csv"
}
```

instead of 
```
path/to/the/csv_file.csv
```
?

I think I get it: the logging is more descriptive this way.